src/server/alternances/domain/alternance.ts

```diff
@@ -6,6 +6,12 @@ export interface Alternance {
 	localisation?: string
 	niveauRequis?: string
 	typeDeContrat?: string
+	source: AlternanceSource
+}
+
+export enum AlternanceSource {
+	MATCHAS = 'matchas',
+	PEJOBS = 'PEJobs',
 }
```

Comment:
```suggestion
export namespace Alternance {
  export enum Source {
	  MATCHA,
	  POLE_EMPLOI,
  }
}
```

Et s'utilise ensuite Alternance.Source. Je ne suis pas sûr qu'on utilise l'index de l'enum 🤔